docs/website/docs/building-from-source/getting-started.md

```diff
@@ -138,32 +144,102 @@ cmake --build ../iree-build/
 
     See also our [developer documentation for ccache](https://github.com/openxla/iree/blob/main/docs/developers/developing_iree/ccache.md).
 
-## What's next?
+### Optional components
+
+By default, the CMake build includes:
+
+* All compiler targets (`llvm-cpu`, `cuda`, `vulkan-spirv`, etc.)
+* All runtime HAL drivers (`local-task`, `cuda`, `vulkan`, etc.)
+* All compiler input formats (StableHLO, TOSA, etc.)
+* All compiler output formats (VM bytecode, C)
```

Comment:
I'm not sure about including this full section here... it's pretty large when scanning through the page. What do others think?

I could also experiment with formatting (e.g. table or task list instead of list).